irs/src/App.js

```diff
@@ -1,18 +1,20 @@
 import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
 import './App.css';
 
 import Login from './components/Login/Login';
 import Register from './components/Registration/Register';
+import Whoyouare from './components/Whoyouare/Whoyouare';
 import MainNav from './components/Navigation/MainNav';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
         <MainNav />
-        <Login />
-        <Register />
-        
+        <Route exact path="/login" component={Login} />
+        <Route exact path="/register" component={Register} />
+        <Route exact path="/whoyouare" component={Whoyouare} />
```

Comment:
good, but make sure to account for the default view of "/" as well-- maybe use a Redirect from react-router-dom here to take the user to the login view if they try to go to "/"